CONTRIBUTING.md

```diff
@@ -189,12 +189,6 @@ notes will look like for the current state of the repo you can run:
 and the release notes in particular will be located at
 `docs/_build/html/release_notes.html`
 
-## Installing Qiskit IBM Experiment from source
```

Comment:
These docs are long deleted. I'm not sure if you want something else to replace this?

Reply:
Hmm, these documents probably need a broader cleanup. Maybe they should recommend tox like qiskit-experiments does.